rio/robot.py

```diff
@@ -47,6 +47,7 @@ def teleopInit(self) -> None:
         # this line or comment it out.
         if self.autonomousCommand is not None:
             self.autonomousCommand.cancel()
+        self.container.robotDrive.imu.getRawGyro()
```

Comment:
What does this line do?

Reply:
I meant to delete that, it was for testing purposes, just forgot to save the file.